### DISCUSSION TITLE
Soap client with certificate autentication

### DISCUSSION THREAD

**Discussion by pedrolopix** [quarkiverse/quarkus-cxf#1009] [Q&A]
Hello!
I'm trying to create a soap client to communicate with the Portuguese tax system, but I'm not getting there. I Saw all the examples but did not figure it out.

Would anyone be able to point me in the right direction?

**Comment by ppalaga** [comment]
Start with https://docs.quarkiverse.io/quarkus-cxf/dev/user-guide/create-project.html and then go on to https://docs.quarkiverse.io/quarkus-cxf/dev/user-guide/first-soap-client.html
